### ISSUE TITLE
💡 [Feature Request] - <title>Make system screen reader accessible?

### ISSUE THREAD

**Issue by nktech1135** [frappe/books#1429]
### Summary

Can frappe books be made screen reader accessible? It's close already, but there are a few issues.

### What problem are you trying to solve?

One example is the dropdowns used inn country and chart of accounts selections in the company setup form. if usin
g a screen reader you can't scroll the list of countries with the arrow keys. The screen reader does not read back the focused item.  Another one i noticed is the check boxes used, the screen reader will not read the checkbox state.
Testing was done using the latest vers
ion of the nvda screen reader from nvaccess.
https://nvaccess.org
I'll be glad to test this if you need feedback.

### Basic Example

make the application usable with a screen reader for blind/visually impaired people

### Drawbacks

from a usability standpoint, none. From a developer standpoint, it's possible we'll need to switch out a few libraries for there more accessible counterparts.
